skabase/SKABaseDevice/SKABaseDevice/SKABaseDevice.py

```diff
@@ -435,18 +435,18 @@ def GetMetrics(self):
     doc_out="The JSON string representing this device, \nfiltered as per the input argument flags", 
```

Comment:
All the XMI files need to be updated to remove these commands: `GetMetrics` and `ToJson`.  And then all the Python files need to be regenerated.

Reply:
XMI files for SKABase and SKAMaster are updated.